### DISCUSSION TITLE
JetBrains GitHub Copilot plugin (1.4.13.4415) new dependency (Git4Idea)

### DISCUSSION THREAD

**Discussion by kss39** [community/community#102748] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hi all,

I am wondering why in the latest JetBrains GitHub Copilot plugin (version 1.4.13.4415) a new dependency was added (Git4Idea). This Git4idea had not been updated for a long time (3 years) and consequently adding this as a dependency will disallow JetBrains Client to install the latest GitHub Copilot plugin.

https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/474473

Thanks!

**Comment by kasparthommen** [comment]
Same question here. This causes problems with the JetBrains IDEs when running in Remote Development mode, where "git4idea" is unavailable.

**Comment by kss39** [comment]
In version [1.4.15.4632] (https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/490424) Git4Idea dependency has been made optional. Thanks for the work!
